Index: setup.py
===================================================================
--- setup.py	(revision 1775)
+++ setup.py	(working copy)
@@ -432,10 +432,10 @@
                       extra_compile_args=[
                         '-IModules/objc',
                       ] + CFLAGS,
-                      extra_link_args=frameworks(
+                      extra_link_args=['-F'+os.path.abspath('Examples/WebKit/UsingWebKitNightlyBuilds')] + list(frameworks(
                         'WebKit',
                         'Foundation'
-                      ),
+                      )),
                       **WebKitDepends
                       ),
         ], headername="WebKit.h")
